File: uri/python/1043.py
A, B, C = [float(x) for x in input().split()]

if abs(B - C) < A < B + C and abs(A - C) < B < A + B and abs(A - B) < C < A + B:
    perimetro = A + B + C
    print('Perimetro = {:.1f}'.format(perimetro))
else:
    area = (A + B) * C / 2
    print('Area = {:.1f}'.format(area))

File: uri/c/1043.c
#include <stdio.h>

float abs(float X, float Y) {
    if (X - Y >= 0) {
        return (X - Y);
    } else {
        return (-1) * (X - Y);
    }
}

int main() {

    float A, B, C;
    scanf("%f %f %f", &A, &B, &C);

    if (abs(B, C) < A && A < (B + C) && abs(A, C) < B && B < A + B && abs(A, B) < C && C < A + B) {
        printf("Perimetro = %.1f\n", A + B + C);
    } else {
        printf("Area = %.1f\n", ((A + B) * C / 2));
    }

    return 0;

}